docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -193,6 +217,15 @@ The following Dagger Function accepts a `Directory` argument, which could refere
       entries
     ```
 
+- Copy the private `user/foo` GitHub repository to `/src` in the container, excluding all Markdown files, and list the contents of the directory:
+
+    ```shell
+    dagger call \
+      copy-directory-with-exclusions --source=ssh://git@github.com/user/foo#main --exclude-file=*.md \
```

Comment:
need to be under single quotes otherwise it is going to be interpreted first by the shell: `--exclude-file='*.md'`